src/CosmosLogStorage.ts

```diff
@@ -0,0 +1,220 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import * as Cosmos from '@azure/cosmos'
+import { CLDebug } from './CLDebug'
+import { ILogStorage, LogQueryResult } from './Memory/ILogStorage'
+import * as CLM from '@conversationlearner/models'
+
+const DATABASE_ID = "LOG_DIALOGS"
```

Comment:
I think it would be better if these properties were be configurable.  Don't want to force users to have `LOG_DIALOGS` db or collection name.  Can't remember how fine grained the permissions were but i think cosmos supported full RBAC like SQL and you could onlly have permissions to manage a collection but not the whole DB so it would be important to allow that flexibility

Reply:
Maybe there was expectation that customers will create their own implementation and change the name in their code. If they end up copying this whole code just to change the name that is a sign that it should be configurable.

Reply:
Agree but this feels like a nice-to-have at the moment